guides/doc-Upgrading_and_Updating/topics/upgrading_a_connected_satellite_server.adoc

```diff
@@ -138,7 +138,7 @@ endif::[]
 ----
 # {foreman-maintain} service restart
 ----
-. If you have migrated content from Pulp2 to Pulp3, remove all old (Pulp2) content.
+. If you have migrated content from Pulp2 to Pulp3, remove all Pulp2 content.
```

Comment:
@jjeffers  can we please have a space between Pulp and the versions also? For example Pulp 2 and Pulp 3? 
Sorry to nitpick at you so much.

Reply:
Ooh! I missed that earlier. No worries, I am grateful for the reviews.